### PR TITLE
Add zsh-autosuggestions + autojump

### PR DESCRIPTION
**Pull request by shreyasbharath**

### CODE REVIEW

File: iTerm/autojump.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# `autojump`
```
Comment (simeg) [inline iTerm/autojump.md]: Love this tool, I use it a lot!

**Comment by simeg** [comment]
Thanks for contributing!

**Comment by shreyasbharath** [comment]
@simeg when do the changes get deployed?

**Comment by simeg** [comment]
@shreyasbharath Deployed!

**Pull request merged**